watchdog/candid.did

```diff
@@ -1,3 +1,19 @@
+type status_code = variant {
```

Comment:
nit: It might be good idea to leave some comments about what the fields represent. Since this is the interface of the canister, it's nice if you can quickly get a high level idea without reading the implementation (e.g. the `other_number` is very hard to know what it is without reading further).

Reply:
Done.